src/wasm/literal.cpp

```diff
@@ -200,33 +215,36 @@ void Literal::getBits(uint8_t (&buf)[16]) const {
       memcpy(buf, &v128, sizeof(v128));
       break;
     case Type::funcref:
-    case Type::nullref:
       break;
     case Type::externref:
     case Type::exnref:
+      if (isNull()) {
+        break;
+      }
```

Comment:
This should probably fall through to the `WASM_UNREACHABLE` since the bits of a null value are still not observable by a Wasm program.

Reply:
Similar to above, in that this previously was

```
    case Type::nullref:	
      break;
```

but with the removal of `nullref` this now becomes a check for `isNull()`. Have been wondering about why it would just return on `nullref` as well, though.

Reply:
Hmm, makes sense. Can we add a TODO about investigating changing this in a follow up?